packages/qttools.rb

```diff
@@ -36,7 +28,6 @@ def self.install
     FileUtils.cp_r 'examples', "#{CREW_DEST_PREFIX}/share/Qt-5"
     FileUtils.cp_r 'include', "#{CREW_DEST_PREFIX}/share/Qt-5"
     FileUtils.cp_r 'mkspecs', "#{CREW_DEST_PREFIX}/share/Qt-5"
-    FileUtils.cp_r 'plugins', "#{CREW_DEST_PREFIX}/share/Qt-5"
```

Comment:
Why no plugins?

Reply:
This folder doesn't exist anymore. It is not part of the tar file and not produced by the compilation step